atomic_reactor/schemas/container.json

```diff
@@ -254,6 +254,31 @@
           "description": "Relative path to directory containing operator manifests",
           "type": "string",
           "pattern": "^[^/].*$"
+        },
+        "repo_replacements": {
```

Comment:
After https://github.com/containerbuildsystem/osbs-client/pull/938, it seems we would also need a change in osbs-client here. TBH, I was actually expecting we would stop checking this schema in atomic reactor.

Reply:
Thanks! I even had the branch ready, just forgot to open the PR.

> TBH, I was actually expecting we would stop checking this schema in atomic reactor.

Yeah same, I think @MartinBasti  already has plans to do something about that.